### DISCUSSION TITLE
UX Feature request: Matching bonus claim button info

### DISCUSSION THREAD

**Discussion by bf-mike-mkt** [betfinio/public#16] [Bugs and Reports]
Right now after claiming my matching bonus, claim button is disabled and shows 0. See picture |
![image](https://github.com/user-attachments/assets/f863ab5a-40a7-4748-86bf-8cb875935caf)

I would like to do these changes to claim button:
1. instead of 0 show how much I can claim tomorow (daily limit, or the rest), for this example there should be 20K now,
2. on mouse hover to this disabled button say: "you can claim this ammount in &14h:02m (variable)"

On mobile view, put alert after clicking this disabled button saying same thing: "you can claim this ammount in &14h:02m (variable)"

**Comment by bf-monster** [comment]
We will discuss this, thank you
